pkg/registration/spoke/spokeagent.go

```diff
@@ -54,7 +59,8 @@ type SpokeAgentConfig struct {
 
 	internalHubConfigValidFunc wait.ConditionWithContextFunc
 
-	hubKubeConfigChecker *hubKubeConfigHealthChecker
+	hubKubeConfigChecker            *hubKubeConfigHealthChecker
+	bootstrapKubeconfigEventHandler *bootstrapKubeconfigEventHandler
```

Comment:
why this is added? I did not see where it is used

Reply:
We got the change from the main branch which was introduced as part of PR #630, but we see in the latest main branch, it's no longer used. We removed it to match the main branch as requested.